README.md

```diff
@@ -56,8 +63,16 @@ cd $HOME/jade
 cp configs/sdkconfig_jade.defaults sdkconfig.defaults
 idf.py flash monitor
 ```
+Use a config file from the configs folder that is specific to your hardware (if available). 
+
+_For example for the TTGO T-Display:_
+```
+cp configs/sdkconfig_display_ttgo_tdisplay.defaults sdkconfig.defaults
+```
+
+If you flash multiple devices or make changes to the original config file that you had used, make sure to delete the `sdkconfig` file that gets created from `sdkconfig.defaults`. Otherwise, your changes do not get picked up when building and flashing the firmware again.
```

Comment:
nit: suggest 'original config file that you used`
nit: suggest `will not get picked up when building and re-flashing the firmware`